powerlevel9k.zsh-theme

```diff
@@ -1500,26 +1500,25 @@ powerlevel9k_prepare_prompts() {
   # Reset start time
   _P9K_TIMER_START=0x7FFFFFFF
 
+  if [[ "$POWERLEVEL9K_RPROMPT_ON_NEWLINE" != true ]]; then
```

Comment:
If this is an impediment for merging, I can add a check for `newline`segments too.
On other hand, if you have `POWERLEVEL9K_PROMPT_ADD_NEWLINE=true` and the right prompt  is printed one line before the left prompt, you can think of the segments in the right prompt as related to the previous command, like the `status` segment. I think this would provide another nice option for the right prompt.

Reply:
This is what I wanted!